designs/shelly/src/base.mjs

```diff
@@ -0,0 +1,209 @@
+    raglanAngle + 180,
+    options.raglanScoopLength * raglanLength
+  )
+
+  // Make the side seams vertical if we're making a tubular shirt. Overrides any hips measurements or options.
+  if (options.straightSides)
+    points.sideHem.x = Math.max(points.armholeCornerScooped.x, points.sideHem.x)
+
+  // Make sure that the shirt at least reaches the armholes, to ensure that the full raglan seam can be formed. This code should only trigger is someone tries to make a really, _really_ short shirt.
+  if (points.sideHem.y < points.armholeCornerScooped.y) {
+    points.sideHem.y = points.armholeCornerScooped.y
+    points.cfHem.y = points.armholeCornerScooped.y
+  }
+
+  const sideAngle = points.sideHem.angle(points.armholeCornerScooped)
+  const sideLength = points.sideHem.dist(points.armholeCornerScooped)
+  points.sideCp1 = points.sideHem
+    .shift(sideAngle, (1 / 3) * sideLength)
+    .shift(sideAngle - 90, options.sideShape * sideLength)
+  points.sideCp2 = points.armholeCornerScooped
+    .shift(sideAngle + 180, (1 / 3) * sideLength)
+    .shift(sideAngle - 90, options.sideShape * sideLength)
+
+  const drawSide = () => {
+    if (options.straightSides)
+      return new Path().move(points.sideHem).line(points.armholeCornerScooped)
+    else
+      return new Path()
+        .move(points.sideHem)
+        .curve(points.sideCp1, points.sideCp2, points.armholeCornerScooped)
+  }
+
+  paths.saBase = drawSide()
+    .curve(points.armholeScoopCp1, points.armholeScoopCp2, points.armholeScoopEnd)
+    .line(points.neckShoulderCorner)
+    .curve(points.neckCP1, points.neckCP2, points.cfNeck)
+    .hide(true)
+
+  paths.foldBase = new Path().move(points.cfNeck).line(points.cfHem).hide(true)
+
+  paths.hemBase = new Path().move(points.cfHem).line(points.sideHem).hide(true)
+
+  paths.seam = paths.saBase.join(paths.foldBase).join(paths.hemBase).close().attr('class', 'fabric')
+
+  if (paperless) {
+    macro('hd', {
+      from: points.cfHem,
+      to: points.sideHem,
+      y: points.sideHem.y + (sa + 15),
+    })
+    macro('vd', {
+      from: points.sideHem,
+      to: points.armholeCornerScooped,
+      x: Math.max(points.sideHem.x, points.armholeCornerScooped.x) + (15 + sa),
+    })
+    macro('vd', {
+      from: points.armholeCornerScooped,
+      to: points.armholeScoopEnd,
+      x: points.armholeCornerScooped.x + (30 + sa),
+    })
+    macro('hd', {
+      from: points.armholeScoopEnd,
+      to: points.armholeCornerScooped,
+      y: 0 - (sa + 0),
+    })
+  }
+
+  if (complete) {
+    points.cutonfoldFrom = points.cfNeck
+    points.cutonfoldTo = points.cfHem
+    macro('cutonfold', {
+      from: points.cutonfoldFrom,
+      to: points.cutonfoldTo,
+      grainline: true,
+    })
+
+    points.title = new Point(
+      points.armholeCorner.x / 2,
+      (points.cfHem.y + points.armholeCornerScooped.y / 2) / 2
+    )
+    macro('title', { at: points.title, nr: 5, title: 'base' })
+
+    points.logo = points.title.shift(-90, 70)
```

Comment:
Instead of using a hardcoded "70 mm", you might instead want to consider calculating a distance based on `points.title.y` and  `points.cfHem.y`. That would guarantee that the logo doesn't fall outside of the part on very small doll si
![Screenshot 2023-05-09 at 7 45 28 PM](https://github.com/freesewing/freesewing/assets/109869956/073e221b-0454-4f99-8051-3e2e52e02059)
zes.